pictures-automation.js

```diff
@@ -0,0 +1,24 @@
+var fs = require('fs');
```

Comment:
I've seen that but this don't go through webpack so I can't use ES6 features.
I can remove the semi-colons ;)